Analysis/TrueJet/src/TrueJet.cc

```diff
@@ -837,6 +848,12 @@ void TrueJet::processEvent( LCEvent * event ) {
         // pre-PS part
 
       LCCollectionVec* fafpi_vec = new LCCollectionVec(LCIO::RECONSTRUCTEDPARTICLE ) ;
+      auto fafpiPidHandler = UTIL::PIDHandler(fafpi_vec);
+      auto fafpi_mainPidId = fafpiPidHandler.addAlgorithm("TrueJet_fafpi", {});
+      std::array<int, 25> fafpi_pidIds{};
+      for (size_t i = 0; i < fafpi_pidIds.size(); ++i) {
+          fafpi_pidIds[i] = fafpiPidHandler.addAlgorithm("TrueJet_fafpi_jet_" + std::to_string(i), {});
```

Comment:
can we format this to left-pad with 0 to have nicer sorting in `podio-dump` etc.?

Reply:
So basically, `TrueJet_fafpi_jet_01` for example?

Reply:
yes

Reply:
done. Should work until someone wants more than 99 of these ;)